shared/common-adapters/text.native.js

```diff
@@ -232,6 +238,19 @@ export const styles = {
     fontSize: 18,
     lineHeight: 24
   },
+  textBadge: {
+    ...textCommon,
+    ...globalStyles.fontBold,
+    lineHeight: 11,
+    fontSize: 11
+  },
+  textBadgeAndroid: {
+    position: 'relative',
+    bottom: 3,
+    marginBottom: -3,
+    paddingBottom: 0,
+    lineHeight: 13
```

Comment:
the text is def off globally. we should fix that asap as we'll have to adjust everything